src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -334,35 +359,63 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     }
   }
 
-  override async fetchTransactions(
-    address: string,
-    nextPage: string | null,
+  override async fetchAccountTransactions(
```

Comment:
We are no longer removing the pending transaction upon fetching it from the none pending transaction endpoints.

Reply:
The adapter is not responsible for doing any state management. Updating the local pending transactions is happening in the `latestTransactionList` as the pending transactions are stored there. Please see: https://github.com/superhero-com/superhero-wallet/pull/2659/files#diff-dbd73d02753926bab2746dfd799dec8c1b05c3920dfb52003d5f02bb3539521dR82-R84

Reply:
I've missed that.